src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -459,7 +480,8 @@ uint32_t PolicyHandler::GetAppIdForSending() const {
     return 0;
   }
 
-  return ChooseRandomAppForPolicyUpdate(apps_with_none_level);
+  app_id = ChooseRandomAppForPolicyUpdate(apps_with_none_level);
```

Comment:
Why this change?

Reply:
Probably as a result of changes that were since removed, I can revert this bit